nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/nf-status-history.js

```diff
@@ -356,8 +356,8 @@
 
             // add status history details
             var detailsContainer = buildDetailsContainer('Status History');
-            Object.keys(statusHistory.details).forEach(function(key) {
-                addDetailItem(detailsContainer, key, statusHistory.details[key]);
+            Object.keys(statusHistory.details).forEach(function(key, value) {
```

Comment:
The method here should be `.entries`.
```suggestion
            Object.entries(statusHistory.details).forEach(function(key, value) {
```